docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -165,21 +63,69 @@ For every new plugin, we perform a manual review that includes the following che
+
+   {{< figure src="/static/img/docs/plugins/plugins-submission-create2.png" class="docs-image--no-shadow" max-width="650px" >}}
+
 1. Enter the information requested by the form.
    - **OS & Architecture:**
      - Select **Single** if your plugin archive contains binaries for multiple architectures.
      - Select **Multiple** if you'd like to submit separate plugin archives for each architecture.
        This can lead to faster downloads since users can select the specific architecture they want to install the plugin on.
    - **URL:** A URL that points to a ZIP archive of your packaged plugin.
-   - **Source Code URL:** A URL that points to a public git repository or ZIP archive of your complete plugin source code.
+   - **Source Code URL:** A URL that points to a public Git repository or ZIP archive of your complete plugin source code. See [examples](#what-source-code-url-formats-are-supported).
    - **MD5:** The MD5 hash of the plugin specified by the **URL**.
 1. Click **Submit**.
+
+## Frequently asked questions
+
+### Do I need to submit a private plugin?
+
+- No. Please only submit plugins that you wish to make publicly available for the Grafana community.
+
+### How long does it take to review my submission?
+
+- We're not able to give an estimate at this time, though we're constantly working on improving the time it takes to review a plugin.
+
+### Can I decide a date when my plugin will be published?
+
+- No. We cannot guarantee specific publishing dates, as plugins are immediately published after a review based on our internal prioritization.
+
+### Can I see metrics of my plugin installs, downloads or usage?
+
+- No. We don't offer this information at the moment to plugin authors.
+
+### How can I update my plugin's catalog page?
+
+- The plugin's catalog page content is extracted from the plugin README file.
+  To update the plugin's catalog page, submit an updated plugin with the new content included in the README file.
+
+### Can I unlist my plugin from the Grafana plugin catalog in case of a bug?
+
+- In the event of a bug, unlisting the plugin from our catalog may be possible in exceptional cases, such as security concerns. However, we don't have control over the specific instances where the plugin is installed.
+
+### Can I distribute my plugin somewhere else other than the Grafana plugin catalog?
+
+- The official method for distributing Grafana plugins is through our catalog. Alternative methods, such as installing private or development plugins on local Grafana instances, are available as per the guidelines provided in [this guide]({{< relref "../../administration/plugin-management#install-plugin-on-local-grafana" >}}).
+
+### Can I still use Angular for my plugin?
+
+- No. We will not accept any new plugin submissions written in angular. For more information, refer to our [Angular support deprecation documentation]({{< relref "../angular_deprecation" >}}).
```

Comment:
Should Angular be capitalized? It is in the heading, so I capitalized the A here. 
```suggestion
- No. We will not accept any new plugin submissions written in Angular. For more information, refer to our [Angular support deprecation documentation]({{< relref "../angular_deprecation" >}}).
```